.env.example

```
@@ -24,3 +24,4 @@ SKIP_ETAG_VALIDATION=true
 ZOOM_ACCOUNT_ID=        # Zoom account ID
 ZOOM_CLIENT_ID=         # OAuth app client ID 
 ZOOM_CLIENT_SECRET=     # OAuth app client secret (keep secure!)
+ZOOM_SECRET_TOKEN=      # Oauth app secret token (for webhook events)
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)